specification/ai/HealthInsights/HealthInsights.OncoPhenotype/tspconfig.yaml

```diff
@@ -32,7 +32,7 @@ options:
     package-name: "{package-dir}"
     emitter-output-dir: "{python-sdk-folder}/sdk/{service-directory-name}/{package-name}"
   "@azure-tools/typespec-java":
-    package-dir: "azure-healthinsights-cancerprofiling"
+    package-dir: "azure-health-insights-cancerprofiling"
```

Comment:
They should match those in SDK repo
https://github.com/Azure/azure-sdk-for-java/tree/main/sdk/healthinsights